osu.Game/Screens/OnlinePlay/OnlinePlayComposite.cs

```diff
@@ -53,5 +57,46 @@ public class OnlinePlayComposite : CompositeDrawable
 
         [Resolved(typeof(Room))]
         protected Bindable<TimeSpan?> Duration { get; private set; }
+
+        /// <summary>
+        /// The currently selected item in the <see cref="RoomSubScreen"/>, or the first item from <see cref="Playlist"/>
+        /// if this <see cref="OnlinePlayComposite"/> is not within a <see cref="RoomSubScreen"/>.
+        /// </summary>
+        protected IBindable<PlaylistItem> SelectedItem => selectedItem;
```

Comment:
Have you looked into having something like `RoomSubScreenComposite` or similar? I'm thinking of splitting this conditional behaviour as described in the xmldoc across two classes, therefore eliminating the conditional-ing (the `if...else` binding and the `CanBeNull` resolution).

I wouldn't be surprised that that's not trivial to achieve, so I'm not going to push for it very strongly. It's just that reading this xmldoc makes me feel like this bindable is a little dicey.